VideoLocker/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -166,6 +167,7 @@ public WebResourceResponse shouldInterceptRequest(WebView view, String url) {
 
             // suppress external links on ZeroRated network
             if (isExternalLink(url)
+                    && !ConfigUtil.isWhiteListedURL(url)
```

Comment:
Is there a good way we can unit test this? Like wrap this logic up into one function and then have a list of URLs and whether they should accept or reject?